source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseWalletDialog.js

```diff
@@ -80,6 +80,12 @@ const messages = defineMessages({
     description:
       'Label for continue button on the delegation setup "choose wallet" step dialog.',
   },
+  syncingWallet: {
+    id: 'staking.delegationSetup.chooseWallet.step.dialog.syncingWallet',
+    defaultMessage: '!!!in sync',
```

Comment:
Wouldn't make more sense to for this copy to be "syncing" instead? "In sync" sounds like it's 100% synced. cc @darko-mijic @nikolaglumac

Reply:
Changed